content/spin/install.md

```diff
@@ -22,7 +22,11 @@ Spin runs on Linux (amd64 and arm64), macOS (Intel and Apple Silicon), and Windo
 
 {{ startTab "Linux"}}
 
-There are multiple ways to install Spin. The easiest is to use the installer script, hosted on this site. The script installs Spin along with a starter set of language templates and plugins:
+You can now manage your Spin installation via [Homebrew](https://brew.sh/). Homebrew automatically installs Spin templates and Spin plugins, and on uninstall, will prompt you to delete the directory where the templates and plugins were downloaded:
```

Comment:
There is a [homebrew core formula for a verification tool called spin](https://formulae.brew.sh/formula/spin). We should call out that a `brew install spin` will not install our Spin. They need to install the spin from our tap. (another reason i recommend breaking out the steps into a tap install and then a project install to emphasize the tap).

Reply:
Thanks, note added.